### PR TITLE
Added libsolace 0.3.8 and 0.3.9 recipe

### PR DESCRIPTION
**Pull request by abbyssoul**
Specify library name and version:  **libsolace/0.3.9**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/libsolace/all/conandata.yml [hunk 1]
```diff
@@ -5,4 +5,10 @@ sources:
   "0.3.7":
     sha256: 7f9790f850a0d7209f95be03e1c58e67f91faeec2e12709893cf32e5f09aa6d0
     url: https://github.com/abbyssoul/libsolace/archive/v0.3.7.zip
+  "0.3.8":
```
Comment (danimtb) [inline recipes/libsolace/all/conandata.yml]: Do you really need the `.8` version or the latest patch `.9` would work?

Comment (abbyssoul) [reply recipes/libsolace/all/conandata.yml]: I realized that I forgot to publish .8
So yeah - .9 should be enough but I thought I should include .8 to fix the omission.

Comment (danimtb) [reply recipes/libsolace/all/conandata.yml]: Don't worry. We try not to package all the patches of the libraries unless it is required by a dependency or specifically requested by the community. thanks!

Comment (danimtb) [reply recipes/libsolace/all/conandata.yml]: @abbyssoul please remove the .8 version if not needed. I will merge the PR then. Thanks!

Comment (abbyssoul) [reply recipes/libsolace/all/conandata.yml]: done.

**Pull request merged**